ci/Jenkinsfile.combined

```
@@ -29,16 +32,24 @@ pipeline {
     stage('Build') {
       parallel {
         stage('Linux/x86_64') { steps { script {
-          linux_x86_64 =   jenkins.Build('status-desktop/systems/linux/x86_64/package')
+          linux_x86_64 = getArtifacts(
+            'Linux', jenkins.Build('status-desktop/systems/linux/x86_64/package')
+          )
```

A bit more readable to my taste:
```suggestion
          linux_x86_64 = jenkins.Build('status-desktop/systems/linux/x86_64/package')
          getArtifacts('Linux', linux_x86_64)
```

---

It's functional, so it must be good :D.